src/external/libusb.BUILD

```diff
@@ -3,25 +3,10 @@
 
 cc_library(
     name = "libusb",
-    srcs = glob([
-        "libusb/*.c",
-        "libusb/os/linux_*.c",
-        "libusb/os/*_posix.c",
-    ]),
-    hdrs = glob([
-        "libusb/*.h",
-        "libusb/os/linux_*.h",
-        "libusb/os/*_posix.h",
-        "Xcode/config.h",
-    ]),
-    includes = [
-        ".",
-        "Xcode",
-        "libusb",
-        "libusb/os",
-    ],
-    deps = [
-        "@libudev",
-    ],
+    srcs = glob(["usr/lib/libusb*.so",
+                 "usr/lib/x86_64-linux-gnu/libusb*.so",
```

Comment:
Isn't this still a Linux-specific path?

Reply:
Yes, libusb remains reliant on platform-dependent binaries, but we can handle multiple platforms using the glob.

Reply:
Hmm, not a huge fan of this implicit way of handling multiple platforms either. I think this will result in nasty build errors if the libraries are not there (as opposed to a nice bazel analysis-time error).

Reply:
It'll only result in linking errors at the end, which is fairly easy to debug IMO. 
Bazel can't distinguish between different Linux platforms (for now), but we can use `select` to choose Linux vs macOS directories which will error out if it encounters an unsupported platform.

Reply:
Yea I'd be more concerned about the Linux vs MacOS case, but sounds like this may be resolvable if you build from source per the comments above.

Reply:
Ended up accepting this after offline discussion.